ui/accordion.info/ui/accordion.reel/accordion.js

```diff
@@ -0,0 +1,152 @@
+/**
+ * @module accordion
+ * @requires montage/ui/montage
+ * @requires montage/ui/component
+ */
+var Montage = require("montage").Montage,
+    Component = require("montage/ui/component").Component;
+
+/**
+ * @class Accordion
+ * @extends Component
+ */
+exports.Accordion = Montage.create(Component, {
+
+    /**
+     * @function prepareForActivationEvents
+     * @param null
+     * @description Add the the event listener for this class object
+     * @private
+     */
+    prepareForActivationEvents: {
+        value: function () {
+            if (window.Touch) {
+                this.element.addEventListener('touchend', this);
+            } else {
+                this.element.addEventListener('mouseup', this);
+            }
+        }
+    },
+
+    /**
+     * @function findClosestOfType
+     * @param el -- Element object
+     * @param type -- Parent flag for element needed find
+     * @param context -- Context for the object
+     * @param clazz -- Flag needed find
+     * @private
+     */
+    _findClosestOfType: {
+        value: function (el, type, context, clazz) {
+            if (el.tagName.toLowerCase() === type) {
+                return el;
+            }
+            var found = false;
+            while (el) {
+                el = el.parentNode;
+                if (el.tagName && el.tagName.toLowerCase() === type) {
+                    if (clazz) {
+                        if (el.classList.contains(clazz)) {
+                            found = true;
+                            break;
+                        }
+                    } else {
+                        found = true;
+                        break;
+                    }
+                }
+                if (!el || !el.ownerDocument || el === context || el.nodeType === 11) {
+                    break;
+                }
+            }
+            if (found) {
+                return el;
+            }
+            return null;
+        }
+    },
+
+    /**
+     *@property headerBackgroundColor
+     *@description Set the background color of header
+     */
+    headerBackgroundColor: {
+        value: null
+    },
+
+    /**
+     * @property title
+     * @description This is the label of header,it is an array.
+     */
+    titles: {
```

Comment:
As I was working on the spec, it occurred to me that it would be much simpler to get the title/label/header from the component being displayed in each section of the accordion. So we wouldn't need that, but eventually a "titleExpression" which is how we would know what frb expresion to use to extract the display name from each component being displayed